Unix/pal/palcommon.h

```diff
@@ -22,7 +22,7 @@
 #    include <windows.h>
 #    include <strsafe.h>
 #else
-#    include "../../common/linux/sal.h"
+#    include "linux/sal.h"
```

Comment:
but quotes are supposed to be relative directories where as brackets are search path.